smartcards/core.py

```diff
@@ -83,6 +86,12 @@ def card_write_short_and_long():
     rv = _write_short_and_long(short_value.encode('utf-8'), long_value.encode('utf-8'))
     return json.dumps({"success": rv})
 
+@app.route('/card/write_long', methods=["POST"])
+def card_write_long():
+    long_value = request.data
+    rv = _write_long(long_value)
```

Comment:
I think we leave as is, because write_long is meant to be for raw bytes, whereas write_short_and_long is being used for JSON election configuration. I suspect we'll want to be much stricter about this soon, but probably ok for now.